test/html-to-react-tests.js

```diff
@@ -340,7 +340,7 @@ describe('Html2React', function () {
                 assert.equal(reactHtml, htmlExpected);
             });
 
-            it('should return false in case of invalid node', function () {
+            it('should return undefined in case of invalid node', function () {
```

Comment:
What I don't understand is why we need to change the public API? I'm not sure why the public API is to return false in the first place, but still can't we fix this bug while retaining the API?